emperor/support_files/js/controller.js

```diff
@@ -13,6 +13,24 @@
  *
  */
 
+
+// Constants
+var gridScale = 0.9;        // Scaling constant for grid dimensions
```

Comment:
These variables seem rather odd, we probably need to change these to look more like constants. And probably scope them specific to the object. Scopes in JavaScript can get to be tricky but are important.